docs/discovery.md

```diff
@@ -37,6 +37,30 @@ class MyClass
 }
 ```
 
+## HTTP Async Client Discovery
+
+This type of discovery finds installed HTTP Async Clients.
```

Comment:
lets say "a HttpAsyncClient" instance. from the point of the user, it only finds one.